runtimes/php/varnish.md

```diff
@@ -2,3 +2,107 @@
 title: Use Varnish as HTTP Cache
 shortdesc: This article shows you how enable Varnish on your application.
 ---
+
+# Varnish
+
+Varnish is a HTTP cache who works as a reverse proxy between the application server and the client, on the server-side.
+Following rules defined by the user, Varnish will caches the data of an applications to reduce the load.
+
+## Enable Varnish for you appliaiton
+
+Varnish is provided by default in PHP instances. To enable it, you just have to create a `varnish.vcl` file
+in the `/clevercloud` folder.
+
+This file describe how Varnish caches your applications and how it decides to return a cached resource or not.
+
+***Warning***: you should not include the `backend` section in your `varnish.vcl` file. We already manage it for you.
+
+## Varnish version / version migration
+
+We provide **Varnish 4**. If you need to upgrade your version of Varnish, you can use script as
+[varnish3to4](https://github.com/fgsch/varnish3to4) to easily upgrade your Varnish version. Do not forget to add the
+following line in top of your Varnish file:
```

Comment:
We use **Varnish 4**. If you already have a configuration file written for varnish 3, you can use [varnish3to4](…) to convert it to varnsih 4. Dont forget to add `vcl 4.0;` at the beginning of the config file.